bindings/go/src/fdb/tuple/tuple.go

```diff
@@ -66,13 +68,60 @@ type TupleElement interface{}
 // packing T (modulo type normalization to []byte, uint64, and int64).
 type Tuple []TupleElement
 
+// String implements the fmt.Stringer interface and returns human-readable
+// string representation of this tuple. For most elements, we use the
+// object's default string representation.
+func (tuple Tuple) String() string {
+	sb := strings.Builder{}
+	printTuple(tuple, &sb)
+	return sb.String()
+}
+
+func printTuple(tuple Tuple, sb *strings.Builder) {
+	// TODO: Add VersionStamp printer
```

Comment:
What actually gets printed for versionstamps now?

Reply:
`%v` format for structures print the value of each fields in format like which in case would be `{[.. <transaction version bytes> ..], <user_version>}`. 

I've just pushed implementation of String() for Versionstamp taking Java implementation as reference -- `Versionstamp(fdb.Printable(<TransactionVersion>), <UserVersion>`)